sql/api/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -99,6 +99,11 @@ abstract class RuntimeConfig {
    */
   private[sql] def get[T](entry: ConfigEntry[T], default: T): T
 
+  /**
+   * Returns whether a particular key is set.
```

Comment:
I think we should probably write a little bit more, e.g., if the configurations are known with default values, it will return true.

Reply:
I am pretty sure it does not.